### PR TITLE
detail what is synced in Active DR

### PR DESCRIPTION
**Pull request by Mossman1215**
after a ticket https://www.cwp.govt.nz/service-desk/requests?target=view.php%3Fid%3D35188
i wanted to detail what is synced in active DR for future developers.

### CODE REVIEW

File: docs/en/03_How_tos/preparing_your_site_for_active_dr.md [hunk 1]
```diff
@@ -6,12 +6,14 @@ summary: How to Prepare your website to use Active DR infrastructure.
...
+Active DR stacks have the following properties:
 
  * They are load-balanced: the traffic is served from two datacentres, increasing the maximum potential capacity of the
  stack.
- * They are highly-available: if one datacentre / node exhibits a problem and is not reachable, or emits 5xx HTTP status
- code, this node will be pulled out of the pool and all traffic will be redirected to the other datacentre.
+ * They are highly-available. If one data centre / node exhibits a problem and is not reachable, or emits 5xx HTTP status
+ codes, this node will be pulled out of the pool and all traffic will be redirected to the other data centre.
+ * The databases on each node is synchronised seamlessly by the infrastructure.
+ * Files are synchronised from the ./www/assets/ directory but no other directories are synchronised between nodes.
```
Comment (NightJar) [inline docs/en/03_How_tos/preparing_your_site_for_active_dr.md]: Is this always true on CWP, or is it possible for SilverStripe 4 to use the `public` folder?
The latter would look like `./www/public/assets`
But I'm not sure of the server configuration to say that would/could ever be the case.

Comment (Mossman1215) [reply docs/en/03_How_tos/preparing_your_site_for_active_dr.md]: I didn't think very hard about the actual on disk location of assets I'll need to do some research to understand how that works and I'll get back to you with an updated commit

Comment (Mossman1215) [reply docs/en/03_How_tos/preparing_your_site_for_active_dr.md]: I had a quick look at our setup and I'll need another PR for the 2.X line to include public webroot
My understanding is following for SS3 & SS4 on cwp
SS3 cwp1.X /sites/<Stackid>/www/assets
SS4 cwp2.X /sites/<Stackid>/www/public/assets

Comment (Mossman1215) [reply docs/en/03_How_tos/preparing_your_site_for_active_dr.md]: Created specific PR for 2.0 onwards
https://github.com/silverstripe/cwp/pull/165

Comment (Mossman1215) [reply docs/en/03_How_tos/preparing_your_site_for_active_dr.md]: As it turns out we just use different sim-links to achieve this

Comment (NightJar) [reply docs/en/03_How_tos/preparing_your_site_for_active_dr.md]: Ideal, thanks for following up :)

**Comment by NightJar** [comment]
All merged up until `1`
#165 will merge up to `master`

**Pull request merged**